schemas/styles/csl-choose.rnc

```diff
@@ -0,0 +1,84 @@
+namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
+namespace cs = "http://purl.org/net/xbiblio/csl"
+
+
+## cs:choose - Conditional Statements"
+div {
+  rendering-element.choose =
+    
+    ## Use to conditionally render rendering elements.
+    element cs:choose { choose.if, choose.else-if*, choose.else? }
+  
+  ## When specifying conditions, we allow either the attribute syntax or the element-based one.
+  choose.condition = condition.atts | condition.elem
+  choose.if =
+    element cs:if { choose.condition+, match, rendering-element* }
+  choose.else-if =
+    element cs:else-if { choose.condition+, match, rendering-element* }
+  choose.else = element cs:else { rendering-element+ }
+  
+  ## This element allows for more complex conditional logic; for 
+  ## example, if you need to specify mixed "match" rules.
+  condition.elem = element cs:condition { match?, condition.atts }
```

Comment:
I think there is usability value in forcing all `condition` elements to be grouped together by making them children of a single `condition` element. That would prevent something like, where a style writer misunderstands that all conditions apply to the single `if` and that `else-if` is needed for other cases (this `if` will always return false because an item cannot have two types):
```
<if>
  <condition type="article-journal"/>
  <text variable="title"/>
  <condition type="book"/>
  <text variable="title" text-case="title" font-face="italic"/>
</if>
```

Or this, where the style will work, but the exact logic is unclear. This will be a problem for others to maintain the style:
```
<if>
  <condition variable="editor" match="none"/>
  <text variable="collection-title" text-case="title" font-face="italic"/>
  <condition variable="editorial-director"/>
  <names variable="editorial-director">
    <label form="verb"/>
    <name form="long"/>
  </names>
</if>
```

It would be better to group all conditions together:
```
<if>
  <conditions>
    <condition variable="editor" match="none"/>
    <condition variable="editorial-director"/>
  </conditions>
  <text variable="collection-title" text-case="title" font-face="italic"/>
  <names variable="editorial-director">
    <label form="verb"/>
    <name form="long"/>
  </names>
</if>
```

Reply:
> I think there is usability value in forcing all `condition` elements to be grouped together

I agree with this 100%, and have no strong opinions about adding a containing element, but have you confirmed the current code doesn't already group the elements together in sequence? 

That was my intention, and I think they do, but I'm having problems with my editors ATM, so haven't tested it.

Reply:
Yes, I think this forces grouping. This [RNC cheatsheet](https://gist.github.com/miikka/b137a8705e88b0b6c164#file-cheatsheet-rnc-L35) says: "if the order of the elements does not matter, use "&" instead of ","" => "," means order is important and all `condition` elements must come first. Right?

Reply:
Yes.

Reply:
I just tested on Atom, and it works as I expected. 

For the first choice after `if` or `if-then`, the only choice is `condition`. 

If you then add a `text` element, `condition` is no longer a choice.

But OTOH, I think if we're in doubt, it's better to err on the side of adding back the `containers` wrapper.

Thoughts before I resolve this?